### PR TITLE
Mta 1617 application profile details

### PR DESCRIPTION
**Pull request by anarnold97**

### CODE REVIEW

File: docs/topics/mta-web-edit-application.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+.Procedure
+
+. In *Migration* view, click *Application Inventory*.
+. Click the row of an application to open a summary of the application's current settings.
++
+.View and edit application
+image::mta-web-app-details-01.png[View and edit application]
+
+. To change application settings, click `edit` icon{icon-edit} to open the application settings, and review the application settings. See procedure above for a list of application settings.
```
Comment (anarnold97) [inline docs/topics/mta-web-edit-application.adoc:23]:
```suggestion
. To change application settings, click `edit` icon {icon-edit} to open the application settings, and review the application settings. See procedure above for a list of application settings.
```

File: docs/topics/images/mta-web-apps-details-01.png
Comment (jmle) [inline docs/topics/images/mta-web-apps-details-01.png:1]: Is there any need to have this image duplicated?

File: docs/topics/mta-web-assessing-apps.adoc [hunk 2]
```diff
@@ -22,18 +22,33 @@ The procedure described below uses a built-in questionnaire for assessing the ri
 +
 [NOTE]
 ====
-Only one application can be assessed at a time.
+You can assess only one application at a time.
 ====
-. Click  *Assess*.
+. In the {ProductShortName} user interface, select the *Migration* working mode.
```
Comment (jmle) [inline docs/topics/mta-web-assessing-apps.adoc:27]: This step is number 3 in the documentation, [but in the image](https://github.com/windup/windup-documentation/blob/d4f525219a0877f4c64d7d3fa9e8c5124d0d7b48/docs/topics/images/mta-web-apps-details.png), the number is 1.

File: docs/topics/mta-web-edit-application.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-edit-application_{context}"]
+= Opening and editing an application
+
+You open and edit an existing application to the *Application Inventory* for changes in the assessment and analysis.
```
Comment (jmle) [inline docs/topics/mta-web-edit-application.adoc:9]: Maybe?
```suggestion
You can open and edit an existing application in the *Application Inventory* for changes in the assessment and analysis.
```

File: docs/topics/mta-web-edit-application.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+* Log into a server running {ProductShortName}
+
+.Procedure
+
+. In *Migration* view, click *Application Inventory*.
+. Click the row of an application to open a summary of the application's current settings.
++
+.View and edit application
+image::mta-web-app-details-01.png[View and edit application]
```
Comment (jmle) [inline docs/topics/mta-web-edit-application.adoc:21]: I think this image is not present. Maybe:
```suggestion
image::mta-web-apps-details.png[View and edit application]
```

File: docs/topics/mta-web-edit-application.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+. To change application settings, click `edit` {edit} to open the application settings, and review the application settings. See procedure above for a list of application settings.
```
Comment (jmle) [inline docs/topics/mta-web-edit-application.adoc:23]:
```suggestion
. To change application settings, click `edit` {edit} to open the application settings and review them. See procedure above for a list of application settings.
```

File: docs/topics/mta-web-review-tab.adoc [hunk 6]
```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-review-tab_{context}"]
+= Application review tab
+
+You can review the settings of any application or archetype settings directly from the list.
```
Comment (jmle) [inline docs/topics/mta-web-review-tab.adoc:9]:
```suggestion
You can review the settings of any application or archetype directly from the list.
```

File: docs/topics/mta-web-review-tab.adoc [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+.Procedure
+
+. In *Migration* view, click *Application inventory*.
+. Select an application, opening a details view from the right edge of the browser window.
++
+// Get image from MTA UI
+image::mta-web-review-tab-02.png[Application review pane]
```
Comment (jmle) [inline docs/topics/mta-web-review-tab.adoc:17]: I can't find this image either

File: docs/topics/images/mta-web-apps-details-01.png
Comment (anarnold97) [reply docs/topics/images/mta-web-apps-details-01.png:1]: Aye, you are right neither are needed

File: docs/topics/mta-web-review-tab.adoc (hunk 7 above)
Comment (anarnold97) [reply docs/topics/mta-web-review-tab.adoc:17]: Will remove image ref

Comment (anarnold97) [inline docs/topics/mta-web-review-tab.adoc:17]:
```suggestion

```

**Pull request closed without merging**